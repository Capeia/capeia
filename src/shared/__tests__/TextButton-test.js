// @flow
/* eslint-env jest, jasmine */

import React from 'react'
import renderer from 'react-test-renderer'

describe('<TextButton>', () => {
  it('renders correctly', () => {
    jest.deepUnmock('../TextButton')
    const Component = require('../TextButton').default
    const tree = renderer.create(<Component />).toJSON()
    expect(tree).toMatchSnapshot()
  })
})
