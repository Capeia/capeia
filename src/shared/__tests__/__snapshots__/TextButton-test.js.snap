// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<TextButton> renders correctly 1`] = `
<span
  className="TextButton "
  onClick={[Function]}
  role="button"
  tabIndex={0}
/>
`;
